### ISSUE TITLE
"BuildInstallers.VS2015.cmd : The system cannot find the file specified" in AppVeyor build

### ISSUE THREAD

**Issue by loop-evgeny** [gitextensions/gitextensions#4407]
**Do you want to request a *feature* or report a *bug*?** Bug (potential)

**What is the current behavior?** The build log for a PR contains an error, though the build still succeeds.

https://ci.appveyor.com/project/gitextensions/gitextensions/build/2.50.90.1499

Line 701:

```
BuildInstallers.VS2015.cmd : The system cannot find the file specified.
At line:2 char:1
+ & Setup\BuildInstallers.$env:IdeVersion.cmd
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (The system cann...file specified.:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
The system cannot find the file specified.
```

**What is the expected behavior?** No errors in the build log

**Comment by gerhardol** [comment]
The problem is in MakeMonoArchive.cmd, inorrect xcopy syntax and has no negative effect
I will include a correction together with a PR related to #4269

**Comment by RussKie** [comment]
The fix will need to go to release/2.51 branch.
We will be deprecating mono on the master

**Comment by gerhardol** [comment]
MakeMonoArchive.cmd could be renamed to MakeZipArchive.cmd and be used for test releases still.

**Comment by RussKie** [comment]
...or MakePortableArchive

**Comment by gerhardol** [comment]
Addressed in #4411
